### DISCUSSION TITLE
The dependency Microsoft.AspNetCore.WebUtilities.2.2.0 is deprecated

### DISCUSSION THREAD

**Discussion by ArcaneSoftware** [hardkoded/puppeteer-sharp#2322] [Q&A]
Recently, I am cleaning up the deprecated packages in my project and find the dependency Microsoft.AspNetCore.WebUtilities.2.2.0 is deprecated. See [Here](https://www.nuget.org/packages/Microsoft.AspNetCore.WebUtilities). Is there any plan to solve it?

**Comment by kblok** [comment]
Removing it :)

**Comment by danports** [comment]
FYI, this dependency has since been un-deprecated and will ship again in .NET 8, so it might not be necessary to keep all of the duplicate status code descriptions that were added in #2324: https://github.com/dotnet/aspnetcore/pull/48265

**Reply by kblok** [reply]
The limitation we have is that pptr# is .netstandard.

**Reply by danports** [reply]
That may also be solved soon enough: https://github.com/dotnet/aspnetcore/issues/50236

**Reply by kblok** [reply]
Awesome! I subscribed to that issue. Thanks!
